services/slam/builtin/builtin_test.go

```diff
@@ -527,7 +529,7 @@ func TestGeneralNew(t *testing.T) {
 		// Create slam service
 		logger := golog.NewTestLogger(t)
 		_, err := createSLAMService(t, attrCfg, logger, false, false)
-		test.That(t, fmt.Sprint(err), test.ShouldContainSubstring, "error with slam service slam process:")
+		test.That(t, fmt.Sprint(err), test.ShouldContainSubstring, "runtime slam service error: invalid slam algorithm \"test\"")
```

Comment:
Yeah I think there was a bug in this test. The original test didnt have a valid camera so it skipped the runtimeServiceValidation check because of this line: https://github.com/viamrobotics/rdk/blob/main/services/slam/builtin/builtin.go#L195. So I added a valid camera and the test get caught because of the invalid algo now